module/data.tf

```diff
@@ -37,7 +37,11 @@ locals {
   # Number master resources to create. Defaults to the number of AZs in the region but should be 1 for regions with odd number of AZs.
   master_resource_count = "${var.force_single_master == 1 ? 1 : length(local.az_names)}"
   # Master AZs is used in the `kops create cluster` command
-  master_azs = "${var.force_single_master == 1 ? element(local.az_names, 0) : join(",", local.az_names)}"
+  odd_number_of_azs = "${length(local.az_names) / 2 == ceil(length(local.az_names) / 2)}"
```

Comment:
Won't `odd_number_of_azs` be 0 when AZ count is 3?

`length(local.az_names) / 2 == ceil(length(local.az_names) / 2)`
is `3/2 == ceil(3 / 2)`
is `1.5 == ceil(1.5)`
is `1.5 == 2`
is `False` or `0`

Or am I wrong?

Reply:
Uff, yes. Embarrassing typo, will fix.